src/JustEat.StatsD/Buffered/Tags/TagsLocation.cs

```diff
@@ -0,0 +1,8 @@
+namespace JustEat.StatsD.Buffered.Tags
+{
+    internal enum TagsLocation
```

Comment:
I think we can just simplify this to a `bool` where it's used - anything different the consumer could just implement the interface themselves.

Reply:
I have changed it by the bool "areBucketNameTags".